addons/l10n_din5008/report/din5008_report.xml

```diff
@@ -103,12 +103,9 @@
                         </tr>
                     </table>
                     <h2>
-                        <span t-if="not o and not docs"><t t-out="company.l10n_din5008_document_title"/></span>
-                        <span t-else="">
-                            <t t-set="o" t-value="docs[0]" t-if="not o" />
```

Comment:
Then is there a reason you remove this line?
It does not seem problematic, and it was added in another commit than the one that added the `company.l10n_de_document_title` (https://github.com/odoo/odoo/commit/d6eb4752d7fbd22dd9ad744db51444e1392fd1bb)

Reply:
Hello @gawa-odoo 
This field is not found in the `res.compnay` object related code, so I remove this line from this template.

Reply:
Yes, that I understood, but why remove the line 108?

Reply:
just changed the sequence when i removed line.